index.html

```diff
@@ -191,14 +187,55 @@ <h2>Uber Navigation</h2>
                 </div>
             </div>
-            <div class="work-section4">
+            <div class="work-section4 firstdev">
                 <img src="image-resources/Snapshoot Portfolio3.jpg" alt="Snapshoot image">
             </div>
         </section>
     </section>
+    <section class="work-main-container container-fluid p-5">
+        <div class="row bg-light work-remain">
+            <span class="closeit"><i class="fa-solid fa-xmark"></i></span>
+            <div class="col-12">
+                <div class="work-main-heading px-2 py-2">
+                    <h1 class="topic-here">Tonic</h1>
+                    <ul class="list-inline flex">
+                        <li class="list-inline-item">CANOPY</li>
+                        <li class="list-inline-item li-first"><i class="fa-solid fa-circle"></i> Back End Dev</li>
+                        <li class="list-inline-item li-first"><i class="fa-solid fa-circle"></i> 2015</li>
+                    </ul>
+                    <img src="image-resources/SPortfolio.jpg" class="img-fluid w-100 " alt="Logo">
+                </div>
+            </div>
+            <div class="col-12">
+                <div class="row d-flex flex-row">
+                    <div class="col-md-8">
+                        <p class="paragh pb-2">
+                            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s.
+                        </p>
+                    </div>
+                    <div class=" col-md-4 d-flex flex-column">
+                        <ul class="list-inline ul-work">
+                            <li class="list-inline-item badge bg-info"><a class= "text-decoration-none text-dark" href="">Html</a></li>
+                            <li class="list-inline- badge bg-info"><a class= "text-decoration-none text-dark" href="">Css</a></li>
+                            <li class="list-inline-item badge bg-info "><a class= "text-decoration-none text-dark" href="">Javascript</a></li>
+                            <ul class="list-inline text-dark">
+                                <li class="list-inline-item badge bg-info"><a class= "text-decoration-none text-dark" href="">Bootstrap</a></li>
+                                <li class="list-inline-item badge bg-info"><a class= "text-decoration-none text-dark" href="">ruby</a></li>
+                                <li class="list-inline-item badge bg-info"><a class= "text-decoration-none text-dark" href="">github</a></li>
+                            </ul>
+                        </ul>
+                        <div class="d-flex flex-row col-md-10 gap-2 pb-2">
```

Comment:
- [ ] In the mobile view, the `See Live` and `See Source` buttons are centralized, but in your design, they are aligned to the left. Kindly fix this alignment issue in the mobile view.